nmoscommon/webapi.py

```diff
@@ -113,9 +113,9 @@ def __cmp(a, b):
         elif b is None:
             return -1
         elif a.priority == b.priority:
-            return cmp(a.accept_index, b.accept_index)
+            return (a.accept_index > b.accept_index) - (a.accept_index < b.accept_index)
         else:
-            return -cmp(a.priority, b.priority)
+            return -((a.accept_index > b.accept_index) - (a.accept_index < b.accept_index))
```

Comment:
Is this right? It seems to have changed a comparison on priority to a comparison on accept_index.

Reply:
Sorry my bad that’s copy and paste engineering